src/app/space/create/workspaces/workspaces-item-actions/workspaces-item-actions.component.less

```diff
@@ -0,0 +1,2 @@
+@import (reference) '../../../../../assets/stylesheets/shared/osio.less';
+.margin-right-20 { margin-right: 20px; }
```

Comment:
This class is not necessary. In `_formulas.less`, I've set the margin variables to be any number from 0-50, so you can just use the class `margin-right-20` and the site will automatically account for it.